content/articles/core-concepts-of-django-to-master/index.md

```diff
@@ -0,0 +1,166 @@
+<!-- Author Biography -->
+Name: ARAFAT OLAWUMI OLAYIWOLA
+I am Software and Associate Cloud Engineer who is passionate about the over-whelming technologies out there. I develop a scalable and user friendly applications that run on all platforms and responsive. I love Django like my girlfriend. Undergraduate of Computer Engineering here in Nigeria.
+
+<!-- My Picture -->
+![Arafat Olayiwola](/core-concepts-of-django-for-mastering/Arafat-Olayiwola.jpg)
+
+![django-img](/core-concepts-of-django-for-mastering/django-image.png)
+
+<!-- INTRODUCTION -->
+Are you  a beginner or an intermediate of Django? Join me to explore more on this superb and sweet framework used to setup web apps by connecting to server. You don't want to miss this!!
+
+### Keys Take Away
+At the end of this article, you will be able to do the following;
+1. Setup Django project and configure it
+2. Have core understanding of how routers work
+3. Know what we mean by MVT in Django world
+4. Understand Django Admin Interface
+5. Be able to authenticate users 
+6. Know what is relational database management system
+
+Let's Begin!!!!
+
+What is Django Framework?
+
+"Django is a free, open source web framework written in the Python programming language". A “web framework” is software that abstracts away many of the common challenges related to building a website, such as connecting to a database, handling security, user accounts and much more. Django uses a python package called pip and this come in handy having installed Python. 
+Django  is used to configure the server side of most website applications as used by Shopify, Google, Pinterest etc. This web framework allows developers out there to develop the backend by actually making ties with frontend of aplications through the use of it concepts, as we want to discuss in this article and databases in a real  world applications. Been a django developer with few years of experience, I have come to conclude on the these concepts as the backbone of Django which you cannot escape if truly you  want to be a succesful Django developer. Below is the bulletin of concepts to be mastered whether you are just starting out learning the framework or you have been into it for a while.
+
+1. Django Project Structure
+2. How Route Works In Web Applications using Django
+3. Understanding Of Model View Template (MVT)
+4. Dajngo Admin Interface Usage
+5. Django User Authentication System
+6. Basics Of Relational Database Management System
+
+<!-- PART ONE -->
+DJANGO PROJECT STRUCTURE
+
+Let me begin by congratulating you for choosing Django as favorite web framework. Now how do we set up Django project in real world?
+Least I forget, Django uses a virtual enviroment. What is virtual enviroment?
```

Comment:
Avoid these rhetorical questions. It usually flows better when you just put a period and go ahead to define the term.